.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Manual Ethflow Refunder",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
```

Comment:
Can we not use a standard image like 

```
"image": "mcr.microsoft.com/devcontainers/typescript-node:1-22-bookworm",
```

Creating our own devcontainer seems a bit over kill